migrations/20220913175746_enabled_to_schedule_enabled.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE mergestat.repo_syncs
+RENAME COLUMN enabled TO schedule_enabled;
+
+COMMIT;
```

Comment:
```suggestion
COMMIT;

```

just adding a linebreak